app/src/ui-blokada/res/values/strings_tunnel.xml

```diff
@@ -16,7 +16,7 @@
     <string name="tunnel_config_wifi_only_title">Download on WiFi only</string>
     <string name="tunnel_config_wifi_only_description">Activate this switch if you wish the app to download lists only when your device is connected to WiFi. Otherwise, the app may also download lists when your device is connected to a mobile network and use your data plan.</string>
     <string name="tunnel_config_watchdog_title">Watchdog</string>
-    <string name="tunnel_config_watchdog_description">Activate this switch if you wish the app to activate the VPN only if there is Internet connectivity, and restart the tunnel in case there are connectivity problems. This may help for some devices. Note: in earlier versions this option was called "Online Only".</string>
```

Comment:
nobody remembers anymore? :D

Reply:
What was the last time is was called "Online Only"? 2.8? :)